src/analytics/pageViewTracker/pageViewTracker.ts

```diff
@@ -20,10 +26,14 @@ export class PageViewTracker extends BaseTracker {
    * @param {PageViewParametersInterface} parameters - The parameters for the page view event.
    * @return {boolean} Returns true if the event was successfully tracked, false otherwise.
    */
+
   trackOnPageLoad(parameters: PageViewParametersInterface): boolean {
     if (this.disableGa4Tracking) {
       return false;
     }
+    if (!this.enablePageViewTracking) {
```

Comment:
It looks like based on this if we set `enablePageViewTracking` to `false`, we exit the function early here so we're also turning form error tracking off?

Reply:
Yeahhh you're right, the trigger for formError Tracker is within pageViewTracker so if that has been disabled, then the formError tracker will be disabled too.  Didn't consider that ....so should I add the same check we have  above for` errorTrigger.length && this.enableFormErrorTracking` but inside this if block too ?